lib/ae_page_objects/elements/collection.rb

```diff
@@ -56,6 +56,13 @@ def configure(options)
       @item_locator = options.delete(:item_locator) || default_item_locator
     end
 
+    def options
+      @options ||= begin
+        evaled_locator = eval_locator(@item_locator).clone
```

Comment:
The intention here was to prevent the original object from being modified when adding the `{ exact: true }` option.  Please have a look at the new version of the code which now uses `dup` instead.